overrides/resources/betterquesting/lang/en_us.lang

```diff
@@ -3387,7 +3387,7 @@ dj2.quest.db.1662.desc=Craft a Ritual Diviner! This handy tool lets you set up a
 dj2.quest.db.1663.title=From a Blood Mage to a Shaman...
 dj2.quest.db.1663.desc=Your first step of becoming a Blood Shaman is to create Rituals that consume your Life Essence for a magical effect!%n%nFor this, you'll need all 4 of these Elemental Inscription Tools. Note that they cannot be use outside of the Ritual Diviner.
 dj2.quest.db.1664.title=Don't get lazy, make a daisy!
-dj2.quest.db.1664.desc=If the daisy%nMakes you crazy,%nBe more hasty%nBiosafety!%n
+dj2.quest.db.1664.desc=If the daisy%nMakes you crazy,%nBe more hasty%nBiosafety!%n%nThe Lexica Botania has much useful information. You can either look through the 10 chapters in the first two rows or use the Lexicon Index to search for keywords throughout all chapters.%nYou can also go to the chapter of an item - in an inventory or in JEI - directly by holding "ctrl" while hovering over the item when having your Lexica Botania in your hotbar. This is possible if the Lexica Botania icon shows to the left of the item name while hovering over it.
```

Comment:
"has much useful information" isnt proper english. "10 chapters on the first two rows" is incorrect terminology - the structure is Category > Entry > Page. there are no chapters. sentence structure is strange, should be reworded